Sources/CSwiftV.swift

```diff
@@ -67,8 +67,8 @@ public class CSwiftV {
     /// - Parameter separator: The string that delimites the cells or fields inside the row. Defaults to ","
     internal static func cells( forRow string: String, separator: String = ",") -> [String] {
         return CSwiftV.split(separator, string: string).map { element in
-            if let first = element.characters.first, let last = element.characters.last , first == "\"" && last == "\"" {
-                let range = element.characters.index(after: element.startIndex) ..< element.characters.index(before: element.endIndex)
+            if let first = element.characters.first, let last = element.characters.last  where first == "\"" && last == "\"" {
+                let range = element.characters.indices.first!..<element.characters.indices.last!
```

Comment:
if the tests still pass it's accurate to the best of my knowledge 👍 however current master is currently on swift 3.0 - would it be possible to revert the 2.2 migration commit? and I'll just take over the addition to the podspec.